docs/pages/versions/v32.0.0/guides/adhoc-builds.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Installing a Custom iOS Expo Client
+---
+
+> **WARNING:** This feature is in alpha release and is not released to production yet.
+
+![Build Overview](/static/images/adhoc-builds-overview.gif)
+
+Build and install a custom version of the Expo Client (link) with your own Apple Credentials using our build service. This custom version of the Expo Client contains features that were previously only available on the Android versions. Our build service will prepare your custom Expo Client, and you can install it to your iOS device directly from our website.
+
+## 0. Prerequisites
+
+You'll need a paid [Apple Developer Account](https://developer.apple.com/programs).
+
+## 1. Install Expo CLI
```

Comment:
I'd bundle the `Install Expo CLI` step into prerequisites and just link to the cli page of docs to be more succinct.